pyvista/ext/coverage.py

```diff
@@ -38,12 +38,12 @@
 
 
 # utility
-def write_header(f: IO, text: str, char: str = '-') -> None:
```

Comment:
This file was excluded from pre-commit checks.  Let's exclude it from local too, unless you think there is value add here.  It looks like a lot of ignores needed to be added.

Reply:
Since the CI check explicitly passes `pyvista` folder, we only will need to exclude the subfolder `ext` from this list to make it the same as before.

https://github.com/pyvista/pyvista/blob/b516e91275eda3260c20a2e1cf0438a7997a6fce/.pre-commit-config.yaml#L55

Reply:
Ok. I'll look into file-level excludes. Note that the current workflow excludes tests, axamples, and examples_trame by default since it's only checking the `pyvista` module/directory (i.e. it runs `mypy pyvista`, not `mypy examples`, etc.)